<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndMain</class>
 <widget class="QMainWindow" name="wndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="wdgCentral">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableWidget" name="twMain">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mbMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actLoad"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuXor_key">
    <property name="title">
     <string>Xor key</string>
    </property>
    <addaction name="actConfigKey"/>
    <addaction name="actHistoryKey"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="menuXor_key"/>
  </widget>
  <widget class="QStatusBar" name="sbMain"/>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actConfigKey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Config key</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actHistoryKey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>History key</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>wndMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
